org.eclipse.buildship.ui/src/main/java/org/eclipse/buildship/ui/internal/util/widget/UiBuilder.java

```diff
@@ -86,6 +86,16 @@ public T control() {
         this.control.setLayoutData(new GridData(SWT.FILL, SWT.CENTER, true, false, 1, 1));
         return this;
     }
+    
+    /**
```

Comment:
Yeah, just use `GridDataFactory`.

Reply:
I've removed all `UiBuilder` references from the `workspace composite` classes, so these changes are useless now